runtime/src/main/java/com/flipkart/flux/config/FileLocator.java

```diff
@@ -42,9 +43,11 @@
 	 */
 	private static final Logger LOGGER = LogManager.getLogger(FileLocator.class);
 	
-	/** The root folder where Flux runtime is deployed*/
-	@Inject
-	private static String deploymentRoot;
+	/** The config root folder of Flux */
+	private static final String DEPLOYED_CONFIG_ROOT;
```

Comment:
This was my confusion too, when I read the code.
We cannot inject a static var, so should I be making deploymentRoot and thus FileLocator injectable?